allensdk/brain_observatory/behavior/stimulus_processing.py

```diff
@@ -77,6 +98,8 @@ def get_images_dict(pkl):
             meta = dict(
                 image_category=cat.decode("utf-8"),
                 image_name=img_name.decode("utf-8"),
+                phase=np.NaN,
+                correct_frequency=np.NaN,
```

Comment:
why the "correct_" prefix?

Reply:
It's called `correct_freq` in the pkl file, for consistency I kept the correct term on the column. That way it's more obvious when looking at the pkl file where it came from.

Reply:
We don't publish our raw files. Even our internal users work through the AllenSDK primarily rather than the raw files, so I don't think we need to worry about trying to name the fields the same. Please name it "frequency" to reduce confusion.

Reply:
Changed to 'frequency'